core/include/esp/gpio.h

```diff
@@ -36,6 +43,19 @@ static inline void gpio_disable(const uint8_t gpio_num)
     *gpio_iomux_reg(gpio_num) &= ~IOMUX_PIN_OUTPUT_ENABLE;
 }
 
+/* Set whether the specified pin continues to drive its output when the ESP8266
+ * goes into sleep mode.  Note that this setting is reset to off whenever
+ * gpio_enable is called, so this must be called after calling that function.
+ */
```

Comment:
I actually don't know..  I also haven't actually verified the `IOMUX_PIN_PULLUP_SLEEP` stuff either.  It's one of those things on my list to look into someday, but I've still never even gotten as far as playing around with the sleep stuff.

I added the function for completeness, though, because otherwise (assuming it does work) there was a way to set it before and isn't anymore (at least using the `gpio_*` routines)..  We can take it out (or annotate it) if it's a concern, though.